src/shared/components/plots/MultipleCategoryBarPlotUtils.ts

```diff
@@ -98,6 +98,71 @@ export function sortDataByCategory<D>(
         }
     });
 }
+export function getSortedMajorCategories(
+    data: IMultipleCategoryBarPlotData[],
+    sortByOption: string | undefined
+): string[] {
+    if (sortByOption === 'SortByTotalSum') {
```

Comment:
see if SortByTotalSum is available on an enum somewhere.  seems likely that it is

Reply:
No, it was not there; I have defined an enumeration and used it